### DISCUSSION TITLE
shouldn't "xx-pagination-*" values also be in response body?

### DISCUSSION THREAD

**Discussion by AwaisKing** [trakt/trakt-api#482] [Q&A]
well, it's just weird to see `xx-pagination` headers and no relevant values in response body
is there any kind of parameter that i should be passing or something?

or if it's not intended, could you guys add that feature? thanks

**Comment by rectifyer** [comment]
That would be a breaking change in the API, so not something we can add to the body response unfortunately. We'll keep this in mind for the next version of the API.

**Reply by AwaisKing** [reply]
i see, and i agree as the response is in JSON Array, but it'd be nice to see it in the next and later versions.
if the `trakt-api-version` header is `>= 3` the structure could be something like:
```
{
  "status": 200,
  "pagination": {
    "current_page": 1,
    "total_pages": 5,
    "total_items": 50,
    "limit": 10
  },
  "data": [
     ...
  ]
}
```
